docs/index.md

```diff
@@ -1,36 +1,50 @@
 ---
 layout: default
-title: What is lakeFS
+title: lakeFS
 description: The lakeFS documentation provides guidance on how to use lakeFS to deliver resilience and manageability to data lakes.
 nav_order: 0
 redirect_from: /downloads.html
 ---
 
-## What is lakeFS
+# Welcome to the Lake! 
+
+<img src="/assets/img/waving-axolotl-transparent.gif" width="90"/>
+
 {: .no_toc }
 
-lakeFS brings software engineering best practices and applies them to data engineering. 
-Concepts such as Dev/Test environments and CI/CD are harder to implement in data engineering, since the data, and not just the code, should be managed.
-lakeFS provides version control over the data lake, and uses Git-like semantics to create and access those versions, so every engineer feels at home with lakeFS in a few minutes.
+**lakeFS brings software engineering best practices and applies them to data engineering.** 
 
-<p class="center">
-    <img src="assets/what_is.png"/>
-</p>
-<p class="center">
-<em>In this reference diagram, lakeFS enables Python applications and Spark jobs with Git-like operations such as branching, committing and rolling back</em>
-</p>
+lakeFS provides version control over the data lake, and uses Git-like semantics to create and access those versions. If you know git, you'll be right at home with lakeFS.
 
-With lakeFS, you can use concepts such as ״branch״ to create an isolated version of the data, ״commit״, to create a reproducible point it time, and “merge” in order to incorporate your changes in one atomic action.
+With lakeFS, you can use concepts on your data lake such as **branch** to create an isolated version of the data, **commit** to create a reproducible point in time, and **merge** in order to incorporate your changes in one atomic action.
 
-lakeFS is an open source project that supports managing data in  AWS S3, Azure Blob Storage, Google Cloud Storage (GCS) and any other object storage with an S3 interface. It integrates seamlessly with popular data frameworks such as Spark, Hive Metastore, dbt, Trino, Presto, and many others and even features an S3 compatibility layer.
+## How Do I Get Started? 
 
-The vision of lakeFS is bringing this functionality across all the data sources in your data pipelines, from analytics databases to key value stores - and to allow one system from which you can easily manage the underlying data in all data stores, with atomic Git-like actions.
+**[The hands-on quickstart](/quickstart) guides you through some of core features of lakeFS**. 
 
+These include [branching](/quickstart/branch.html), [merging](quickstart/commit-and-merge.html), and [rolling back changes](quickstart/rollback.html) to data. 
 
```

Comment:
Maybe add a line here about the installation free cloud environment - which allows you to play around with a sample repo without installing anything.